src/lib/services/scheduler-service.ts

```diff
@@ -0,0 +1,11 @@
+export default class SchedulerService {
+    private intervalIds: NodeJS.Timer[] = [];
+
+    schedule(scheduledFunction: () => void, timeMs: number): void {
+        this.intervalIds.push(setInterval(scheduledFunction, timeMs).unref());
```

Comment:
In the future I can imagine we will improve this implementation to not create a new interval per background task, but rather run at a regular interval and pick tasks in that interval and run them in sequence. I think that would use database connections more efficiently.

Reply:
Good idea. Once we have more code schduled externally some task collapsing would work really nice.